src/app/clusters/air-quality-server/air-quality-server.cpp

```diff
@@ -115,6 +115,9 @@ CHIP_ERROR Instance::Read(const ConcreteReadAttributePath & aPath, AttributeValu
     case Attributes::AirQuality::Id:
         ReturnErrorOnFailure(aEncoder.Encode(mAirQuality));
         break;
+    case Attributes::FeatureMap::Id:
```

Comment:
This should have gotten added to the list of "attribute access interface only" attributes....

Reply:
Looks like it _is_ there, but because of https://github.com/project-chip/zap/issues/543 that annotation is not working right now for global attributes.